src/saveload/saveload.h

```diff
@@ -10,6 +10,7 @@
 #ifndef SAVELOAD_H
 #define SAVELOAD_H
 
+#include "saveload_error.hpp"
 #include "../fileio_type.h"
 #include "../fios.h"
 #include "../strings_type.h"
```

Comment:
saveload_error.h includes strings_type, so it's not needed here anymore.

Reply:
Even though this header itself uses StringID?

Reply:
https://github.com/include-what-you-use/include-what-you-use ?

Reply:
The question is a bit... where to draw the line? Technically this file should then also include `bitmath_func.hpp`, `cassert`, 'memory' and `stdafx.h`.
All in all it doesn't really matter. It's just that less includes help a tiny bit in building performance. Either way is ultimately fine by me.